### PR TITLE
Clipboard Item updates

### PR DESCRIPTION
**Pull request by Rumyra**
A checklist to help your pull request get merged faster:
- [x] Summarize your changes

Adding data to reflect current spec https://www.w3.org/TR/clipboard-apis/#clipboard-interface

* New members added -> presentationStyle will have MDN page https://github.com/mdn/content/pull/2829
* Updating chrome implementation to reflect this issue https://bugs.chromium.org/p/chromium/issues/detail?id=1014310

- [x] Data: link to resources that verify support information (such as browser's docs, changelogs, source control, bug trackers, and tests)

https://bugs.chromium.org/p/chromium/issues/detail?id=1014310
https://github.com/mdn/content/issues/2514

- [x] Data: if you tested something, describe how you tested with details like browser and version

Testing code in Chrome 90 (Canary), FF 87 and Safari 14

- [x] Review the results of the linter and fix problems reported (If you need help, please ask in a comment!)

Tests pass ☑️

- [x] Link to related issues or pull requests, if any

As above

### CODE REVIEW

File: api/ClipboardItem.json [hunk 1]
```diff
@@ -67,10 +67,14 @@
           "description": "<code>ClipboardItem()</code> constructor",
           "support": {
             "chrome": {
-              "version_added": "66"
+              "version_added": "66",
+              "partial_implementation": true,
+              "notes": "The <code>ClipboardItem</code> constructor only accepts a blob as the item data. Full implementation would also allow for a string or a Promise which resolves with either a string or blob. See <a href='https://crbug.com/1014310'>bug 1014310</a>."
```
Comment (ddbeck) [inline api/ClipboardItem.json:72]: If this is ever fixed, the note won't age very well. Suggesting a rewrite to make it easier to prepend "Before version X" to the note, when the time comes. What do you think of this version?

```suggestion
              "notes": "The <code>ClipboardItem</code> constructor only accepts a blob as the item data, but not strings or Promises that resolve to strings or blobs. See <a href='https://crbug.com/1014310'>bug 1014310</a>."
```

File: api/ClipboardItem.json [hunk 2]
```diff
@@ -67,10 +67,14 @@
...
             },
             "chrome_android": {
-              "version_added": "66"
+              "version_added": "66",
+              "partial_implementation": true,
+              "notes": "The <code>ClipboardItem</code> constructor only accepts a blob as the item data. Full implementation would also allow for a string or a Promise which resolves with either a string or blob. See <a href='https://crbug.com/1014310'>bug 1014310</a>."
             },
             "edge": {
               "version_added": "≤79"
```
Comment (ddbeck) [inline api/ClipboardItem.json:80]: The note on Chrome should probably be carried over to Edge, Opera, and Samsung Internet, too

File: api/ClipboardItem.json [hunk 3]
```diff
@@ -124,6 +130,100 @@
           }
         }
       },
+      "createDelayed": {
```
Comment (ddbeck) [inline api/ClipboardItem.json:133]: I'd drop the new features in this PR. None of the three new features are listed as supported by any browser. Unless we have evidence that implementation is imminent, I think it's too soon to add it to BCD. After all, there's no compat risk if something has never been supported by any browser.

**Review by ddbeck — CHANGES_REQUESTED** [review]
Hi, @Rumyra. In line comments, I've proposed a couple of changes. Thank you for the PR!

File: api/ClipboardItem.json (hunk 1 above)
Comment (Rumyra) [reply api/ClipboardItem.json]: Absolutely - I hadn't considered that - learning all the time ☺️

**Comment by Rumyra** [comment]
Thanks again @ddbeck - updated as per feedback ☺️

**Review by ddbeck — APPROVED** [review]
Great. This can go out with today's release. Thank you! 🎉

**Comment by Rumyra** [comment]
\o/ thank you @ddbeck ! I'm v happy I only revised this once 😆 Very appreciative of the feedback 👍

**Comment by ddbeck** [comment]
@Rumyra no problem! And there's lots to learn about BCD. The revisions aren't a sign that something is _wrong_, just that this stuff is complicated.

**Pull request merged**